### ISSUE TITLE
Non-documented prerequisites

### ISSUE THREAD

**Issue by Jaxom99** [PhilBri/Node-Linky#4]
Hi,
Thanks for all this work.
For a quicker onboarding, you could specify the non-trivially installed dependancies. I had to dig through logs (error message was too short)to see that I missed two of them :
- Superagent
- moment.js
On a Debian Stretch fresh installation of node and npm.

**Comment by vdoucet** [comment]
Same comment, as Jaxom99 ... some dependancies are missing ...

**Comment by Jaxom99** [comment]
@vdoucet are they the same ? What is your config ? If we identify the same problem of different platforms, then I could make a simple PR to update the readme accordingly.

**Comment by cedrep** [comment]
I have an issue, and I can't use the node:

Error: Cannot find module 'superagent' (line:4)

**Comment by Jaxom99** [comment]
> I have an issue, and I can't use the node:
> 
> Error: Cannot find module 'superagent' (line:4)

Yeah, as I explained, they are dependancies but are not installed by default. You have to search on Internet how to add them to your Node-Red install.
If someone has the ability to make a PR about it, that would be great ! I still don't know how to do this myself...
